src/components/filter/filter.css

```diff
@@ -0,0 +1,38 @@
+@import url('../../routes/productpage/productPage.css');
+.filter-header{
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+}
+.clear-btn{
+    text-decoration: none;
+    font-size: 2.4rem;
+    font-weight: bold;
+    color: var(--secondary-text-color);
+}
+.clear-btn:hover{
+    color: var(--secondary-color);
+}
+.sort-by-category{
+    display: flex;
+    gap:2rem;
+}
+.filter-body-container .sort-by-price {
+    display: flex;
+    gap:2rem;
+}
+.sort-body-container,.category-body-container,.filtertype-body-container{
```

Comment:
Nice that you have used the basics of CSS and not repeated the code for each class.

Reply:
> All-in-all the code is clean!! Liked it👍

Thanks